tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -29,53 +34,34 @@ spec:
     workingDir: $(workspaces.source.path)
     args: ["clone", "$(params.giturl)"]
   - name: prepare-loadtest
-    image: amazon/aws-cli
+    image: alpine/k8s:1.22.6
     workingDir: $(workspaces.source.path)
     script: |
       cat > "$(workspaces.source.path)/overrides.yaml" <<EOL
       NODES_PER_NAMESPACE: $(params.nodes-per-namespace)
       CL2_LOAD_TEST_THROUGHPUT: $(params.cl2-load-test-throughput)
+      CL2_SCHEDULER_THROUGHPUT_THRESHOLD: 20
       PODS_PER_NODE: $(params.pods-per-node)
       CL2_USE_HOST_NETWORK_PODS: false
       # we are not testing statefulsets at this point
       SMALL_STATEFUL_SETS_PER_NAMESPACE: 0
       MEDIUM_STATEFUL_SETS_PER_NAMESPACE: 0
       # we are not testing PVS at this point
       CL2_ENABLE_PVS: false
-      PROMETHEUS_SCRAPE_APISERVER_ONLY: true
-      PROMETHEUS_SCRAPE_KUBE_PROXY: false
+      PROMETHEUS_SCRAPE_KUBE_PROXY: true
       ENABLE_SYSTEM_POD_METRICS: false
       NODE_MODE: master 
       EOL
       cat $(workspaces.source.path)/overrides.yaml
       cp $(workspaces.source.path)/overrides.yaml $(workspaces.results.path)/overrides.yaml
-  - name: validate-cluster
-    image: amazon/aws-cli
-    workingDir: $(workspaces.config.path)
-    script: |
-      mkdir -p /root/.kube/
-      cp $(workspaces.config.path)/kubeconfig /root/.kube/config
-      # TODO: Move to a separate task and chain it to this task through pipeline if we need more checks than just these before kicking off test.
-      #kubectl commands are purely for knowing state of cluster before kicking off the test.
-      curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
-      install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
-      kubectl version
-      kubectl config current-context
-      kubectl get nodes 
-      kubectl get ns
-      # end
   - name: run-loadtest
-    image: 197575167141.dkr.ecr.us-west-2.amazonaws.com/clusterloader2:76e3fd7
+    image: public.ecr.aws/kit/clusterloader2:latest
```

Comment:
curious, are we not using the tagged version?

Reply:
did you just push this tag  `v0.0.1` yesterday ?  I didn't see any meaningful tags before on that repo.